src/Select/StyledSelect.js

```diff
@@ -7,6 +7,12 @@ import StyledInput from '../Input/StyledInput';
 
 const StyledSelectInput = styled(createWithComponent(StyledInput, 'select'))`
   appearance: none;
+  background: ${props => props.theme.colors.white};
```

Comment:
Should this use `theme.colors.background.white`?

Reply:
Yes,  yes it should.